Stormshield/sns/_meta/fields.yml

```diff
@@ -0,0 +1,44 @@
+stormshield.fw:
+  description: The firewal ID, set by the administrator or by default its serial number
+  name: stormshield.fw
+  type: keyword
```

Comment:
Shouldn’t you use [`observer.serial_number`](https://www.elastic.co/guide/en/ecs/current/ecs-observer.html#field-observer-serial-number) instead?